infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -0,0 +1,34 @@
+server {
+    listen 80;
+    return 301 https://$host$request_uri;
+}
+
+server {
+    listen 443;
+    server_name {{ inventory_hostname }};
+
+    ssl_certificate           {{ nginx_ssl_certificate }};
+    ssl_certificate_key       {{ nginx_ssl_certificate_key }};
+
+    ssl on;
+    ssl_session_cache  builtin:1000  shared:SSL:10m;
+    ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers HIGH:!aNULL:!eNULL:!EXPORT:!CAMELLIA:!DES:!MD5:!PSK:!RC4;
+    ssl_prefer_server_ciphers on;
```

Comment:
I don't know if this cipher configuration is better than the one I proposed here:
https://raw.githubusercontent.com/triplea-game/dice-server/master/config/nginx/dice.tripleawarclub.org
But I think we should agree on a specific set of ciphers we consider as secure.
The ciphers there were chosen in order to pass the qualys ssl labs test a year ago or something. I'm not sure if something changed in the meantime.

Reply:
@RoiEXLab let's follow-up post-merge? IMO it'll be easier once we take some next steps to install a signed cert. Http-client that connects to this server current has cert validation turned off, getting a cert installed and testing different configurations with cert validation I think will make this easier.